File: evidence/generated-systems/angular/healthcare-on-angular/src/app/models/ImagingOrder.js

// Define collection and schema for ImagingOrder
export interface ImagingOrder {
    bodySite:
	type : string
    contrast:
	type : boolean
    Order:
	type : Schema.Types.ObjectId
    ImagingCenter:
	type : Schema.Types.ObjectId
    Reports:
 	type : [{ type: Schema.Types.ObjectId, ref: 'ImagingReport' }]
    Modality:
 	type : String
#
    collection: 'imagingOrders'
}

File: evidence/generated-systems/rubyonrails/healthcare-on-rails/healthcareonrails/app/models/imagingorder.rb
class ImagingOrder < ApplicationRecord
  enum Modality: [:XRay, :CT, :MRI, :Ultrasound, :PET, :Mammography]


  has_many :Order, class_name: 'ClinicalOrder'
  has_many :ImagingCenter, class_name: 'ImagingCenter'
  has_many :Reports, class_name: 'ImagingReport'

end
